android/build.gradle

```
@@ -25,7 +25,16 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 35
```

I think you should add a namespace to this line.
```suggestion
    namespace 'pl.pszklarska.beacon_broadcast'
    compileSdkVersion 35
```